### ISSUE TITLE
Changing KPMs after assessments

### ISSUE THREAD

**Issue by koay9f** [NREL/NEB-Tool#240]
KPMs will change from year to year (especially if we are reducing them via NEBs)
But users may want to return to an old assessment to review
Need a way to "lock down" an assessment so KPM and other data is no longer updated from company/facility data and can no longer update the company/facility data

**Comment by RLiNREL** [comment]
Two potential solutions:
- Like Turbo tax, the previous year tax return can be exported and imported to current year, previous assessment can be "locked" in an exported file, when user want to import back (this will create a duplicate entry, like company name, facility name, etc.)
- Involve with some backend arrangements, create a historical assessment table that stores all locked assessments. We can provide a "lock" button, when user clicks, the assessment goes to that table. To view these assessments, we can provide a dedicated tab for viewing the key inputs or reports only (read-only).
- Combination of above two

**Comment by RLiNREL** [comment]
push back to after board demo

**Comment by rmroot** [comment]
pulling out of KPM calculators epic and closing epic

**Comment by koay9f** [comment]
I think a conversation with the contact from Lion Elastomer will help us flesh this out.  I think this can be put back to be dealt with after release ("version 2.0" type work)

**Comment by koay9f** [comment]
see #621 

maybe "archive" an assessment - it would copy all the pre-visit data to a "new" facility and users could access old versions as if they were other companies?

**Comment by RLiNREL** [comment]
> see [#621](https://github.com/NREL/NEB-Tool/issues/621)
> 
> maybe "archive" an assessment - it would copy all the pre-visit data to a "new" facility and users could access old versions as if they were other companies?

I agree. The easier/cleaner way is to take a snapshot of all facility data (assessment, KPMs, utility settings, etc.). Will follow up on the other issue

**Comment by RLiNREL** [comment]
Closing this one, focusing on the #621
